### ISSUE TITLE
Chroot Broken

### ISSUE THREAD

**Issue by thed3x7er500** [dnschneid/crouton#5027]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial . . . 
crouton: version 1-20221126232257~master : ae594340
release: xenial
architecture: amd64
targets: xfce
host: version 15572.63.0 (Official Build) stable-channel octopus
kernel: Linux Localhost 4.14.254-20843-g176a4ce1ad3f #1 SMP PREEMPT Mon Oct 2 18:31:45 PDT 2023 x86_64 x86_64 x86_64 GNU/Linux
freon: yes

pictures

Unmounting /mnt/stateful_partition/crouton/chroots/xenial . . . 
![output](https://github.com/dnschneid/crouton/assets/110211674/4f57c5f0-c38e-46c5-9523-b89bb74ce79f)
![20231024_085721](https://github.com/dnschneid/crouton/assets/110211674/0d24e455-ade4-4b00-baea-e571deb73ab6)
![20231024_085716](https://github.com/dnschneid/crouton/assets/110211674/655aac66-c5c7-47d2-b6f3-5050b2a8e3e2)

#### Please describe your issue:

my Chroot is broken and it says that i need to update or complete setup using apt-update, but then i type apt-update it says Command not found.

#### If known, describe the steps to reproduce the issue:

for more info:
Unblocked School computer, 
phaser360, version 117.15572.0
Lenovo 300e 2nd gen MTK
proccesser: Intel Celeron
Board name: Octopus

**Comment by CroutonIsFun** [comment]
You now must use the silence branch:

`sudo CROUTON_BRANCH=silence crouton -r xenial - xfce`

This will prevent crouton from attempting to compile cras, as the 3rd-party tool called autotools has been removed from the repositories.

You now need to delete your chroot and start over.

**Comment by thed3x7er500** [comment]
thank you, but now its not letting me run sudo startxfce4 from VT-2. i cant use Sudo in crosh 
![20231026_171149](https://github.com/dnschneid/crouton/assets/110211674/81a57904-40e7-4c5c-a605-4970ccd07ed5)

**Comment by CroutonIsFun** [comment]
Right... You can use sudo in VT-2, but then you can't launch the desktop. VT-2 is based on FreCon, and does not allow a graphics session. So inside your chroot, you need to install ssh, and once it's installed, activate it with 

`sudo /etc/init.d/ssh start`

After all this, go over to the crosh shell, and ssh into localhost with

`ssh foo@localhost`

Once you get inside your chroot, you can launch `startxfce4`

I know it sounds convoluted, but it's easy once you've done it a few times.

**Comment by thed3x7er500** [comment]
got sudo to run in crosh, thanks

**Comment by thed3x7er500** [comment]
![20231127_090423](https://github.com/dnschneid/crouton/assets/110211674/abc69b9b-b24d-4ddc-a8ed-bd4956f22f5c)
![20231127_090413](https://github.com/dnschneid/crouton/assets/110211674/f47c2880-a941-42ab-ae0c-ecfd2e78d042)
![20231127_090345 (2)](https://github.com/dnschneid/crouton/assets/110211674/e798279c-4112-4654-bd82-6d27ee3cb437)

**Comment by CroutonIsFun** [comment]
Your close, but not there yet. From VT-2, you have to install SSH. So go,

`sudo enter-chroot`

Now, inside your chroot, go

`sudo apt install ssh`

The above install only has to be done once. When the installation completes, go

`sudo /etc/init.d/ssh start`

Every time you restart your Chromebook, you will need to login to VT-2, go `sudo enter-chroot` followed by `sudo /etc/init.d/ssh start`

Finally, go over to crosh, and go `ssh foo@localhost`

The first time you do this, you will have to type yes, you want to add this connection. 

Now that you're inside the chroot from crosh, only now, you can go,

`sudo startxfce4`

These are the steps. You need to understand how and why this works. It's currently the only known way to launch a crouton desktop.

**Comment by OddbyteWasTaken** [comment]
if you want to run sudo in crosh, here is a easy tutorial:
https://github.com/OddbyteWasTaken/howto-use-sudo-in-crosh/blob/main/README.md#how-to-use-sudo-in-crosh
